src/assets/variables.scss

```diff
@@ -47,3 +47,11 @@ $opacity_full: 1;
 // menu round background hover feedback
 // good looking on dark AND white bg
 $action-background-hover: rgba(127, 127, 127, .25);
+
+// various structure data
```

Comment:
ahah, please add proper comments like the one above :stuck_out_tongue_closed_eyes: